solutions/swb-reference/jest.config.js

```diff
@@ -48,6 +48,11 @@ async function init() {
     mainAccountId,
     runId: `${Date.now()}`
   };
+
+  if (process.env.LAMBDA_URL) {
```

Comment:
this allows us to point to a local docker environment when running integration tests. you pass in the value as `LAMBDA_URL=whateverURL` when invoking the test the way you put `STAGE=dev`